hypothesis-python/src/hypothesis/internal/floats.py

```diff
@@ -19,7 +19,51 @@
 
 import math
 
-from hypothesis.internal.compat import struct_pack, struct_unpack
+from hypothesis.internal.compat import CAN_PACK_HALF_FLOAT, quiet_raise, \
+    struct_pack, struct_unpack
+
+try:
+    import numpy
+except ImportError:
+    numpy = None
+
+
+STRUCT_FORMATS = {
+    16: (b'!H', b'!e'),  # Note: 'e' is new in Python 3.6, so we have helpers
+    32: (b'!I', b'!f'),
+    64: (b'!Q', b'!d'),
+}
+
+
+# There are two versions of this: the one that uses Numpy to support older
+# versions of Python, and the elegant one for new versions.  We use the new
```

Comment:
nit: Would nice to be specific about what versions of Python count as "older" here.